<img src="./unal.png" align=left width="150" height="150"></img>

$\;$
---


<h2 align="center">PRUEBA DE MEDIAS (muestras grandes)
    <h3 align="center">"prueba dos poblaciones"

    
### Oscar Julian Layton 
---
    
$\;$



Para realizar una prueba de de media para dos muestras se tiene:

## Inferencia de medias para dos muestras independientes

Considerese:

* $X_{11}, X_{12}, X_{13},...,X_{1 n_1}$ una _muestra aleatoria_ de una población distribuida Normal con media $\mu_1$ y varianza $\sigma^2_1$.
* $X_{21}, X_{22}, X_{23},...,X_{2 n_1}$ una _muestra aleatoria_ de una población distribuida Normal con media $\mu_2$ y varianza $\sigma^2_2$.
* Las muestras anteriores son independientes entre si.

$$E[\overline{X}_1 - \overline{X}_2 ]= E[\overline{X}_1] - E[\overline{X}_2] = \mu_1 - \mu_2$$

Ahora considerando que las muestras son independientes  :

$$V(\overline{X}_1 - \overline{X}_2 )=V(\overline{X}_1 ) + V(\overline{X}_2 ) = \frac{\sigma^2_1}{ n_1}  +  \frac{\sigma^2_2}{ n_2} $$

Como las distribuciones poblacionales son Normales, tanto $\overline{X}_1$ como $\overline{X}_2$ son normales, por lo tanto $ \overline{X}_1 - \overline{X}_2 $ es Normal, luego:


$$ \overline{X}_1 - \overline{X}_2  \;\; \sim \;\; N \left ( \mu_1 - \mu_2 , \sqrt{\frac{S^2_1}{ n_1}  +  \frac{S^2_2}{ n_2}} \right ) $$




<div class="alert alert-success">

En consecuencia, al ser n grande por el **teorema central del límite (TCL)**

\begin{equation}
Z=\frac{(\overline{X}_1-\overline{X}_2)-\Delta_o }{ \sqrt{\frac{S^2_1}{ n_1}  +  \frac{S^2_2}{ n_2}} }\hspace{0.4cm} \hspace{0.4cm}  \hspace{1.2cm} ;  \hspace{1.2cm} Z \mid _{H_0} \sim N(0,1)
\end{equation}


</div>


#### 1. Parámetro Poblacional

El parámetro poblacional a contrastar es $\mu_1 = \mu_2$ "Igualdad de medias poblacionales"

#### 2. Hipótesis

Considérese las medias poblacionales $\mu_1$ y $\mu_2$


\begin{equation}
H_{0}: \mu_1 - \mu_2 = \Delta_o \hspace{0.9cm} vs \hspace{0.9cm} H_{1}: \mu_1 - \mu_2 > \Delta_o
\end{equation}

#### 3. Estadística de prueba

$\;$

\begin{equation}
Z=\frac{(\overline{X}_1-\overline{X}_2)-\Delta_o }{ \sqrt{\frac{S^2_1}{ n_1}  +  \frac{S^2_2}{ n_2}} }\hspace{0.4cm} \hspace{0.4cm}  \hspace{1.2cm} ;  \hspace{1.2cm} Z \mid _{H_0} \sim N(0,1)
\end{equation}


#### 4. Construcción regla de desición

Para un $0<\alpha<1$ nivel de significancia

$$P_{\Delta_{o}}(Error\; tipo \; I)=\alpha$$
$$P_{\Delta_{o}}(Rechaz\;H_o\;cuando \;es \;verdadera)=\alpha$$
$$P_{\Delta_{o}}(Z \geq Z_{(1-\alpha)} \; cuando \; N(0,1)) =\alpha $$

En consecuencia:

$$P_{\Delta_{o}}\left (   \frac{(\overline{X}_1-\overline{X}_2)-\Delta_o }{\sqrt{\frac{S^2_1}{ n_1}  +  \frac{S^2_2}{ n_2}}}\geq Z(0,1)   \right )=\alpha$$

#### 5. Regla de Desición


<div class="alert alert-danger">

Para un $\alpha$ de significancia se procede a:

\begin{equation}
\vartheta :  \; \;Rechazar \; H_o  \;\;\;\; sii \;\; \;\; Z \geq Z_{(1-\alpha)}
\end{equation}
</div>

---

Para la **potencia de la prueba** se usa el error tipo II

$$\beta(\mu')=P(Error\; tipo \; II) $$

$$\beta(\mu')= P(Z < Z_{(1-\alpha)} \;\; cuando \; \;\mu_1 - \mu_2 =\Delta^{´})$$


$$\beta(\mu') = P \left (   \overline{X}_1 -\overline{X}_2  < \Delta_o + Z_{(1-\alpha)}(\sqrt{\frac{S^2_1}{ n_1}  +  \frac{S^2_2}{ n_2}}) \;\; con \;\; \mu_1 - \mu_2 =\Delta^{´}   \right )$$

$$\beta(\mu') =  \left (    \frac{(\overline{X}_1-\overline{X}_2)-\Delta^{´} }{\sqrt{\frac{S^2_1}{ n_1}  +  \frac{S^2_2}{ n_2}}} <Z_{(1-\alpha)} + \frac{\Delta_o- \Delta^{´}}{\sqrt{\frac{S^2_1}{ n_1}  +  \frac{S^2_2}{ n_2}} } \right )$$
$$potencia = 1 - \beta$$

----
## Ejemplo (Prueba cola izquierda)


Considerese el datset **births** del paquete openintro el cual contiene información sobre 150 nacimientos junto con información de las madres (1.fumadores - 2.no fumadoras)--> (smoker - nonsmoker). Pruebese si el peso de los niños nacidos con madres que fuman difiere del peso de los niños cuya madre no fuma.

In [10]:
library(openintro)
library(tidyverse)
data(births)
head(births, 4)

f_age m_age weeks premature visits gained weight sex_baby smoke    
1 31    30    39    full term 13      1     6.88   male     smoker   
2 34    36    39    full term  5     35     7.69   male     nonsmoker
3 36    35    40    full term 12     29     8.88   male     nonsmoker
4 41    40    40    full term 13     30     9.00   female   nonsmoker

In [11]:
library(dplyr)

smoker <- filter(births, smoke == "smoker")
nonsmoker  <- filter(births, smoke == "nonsmoker")

#mean(nonsmoker$weight)-mean(smoker$weight)

miremos la distribución de las dos poblaciones.

In [12]:
shapiro.test(smoker$weight)
shapiro.test(nonsmoker$weight)


	Shapiro-Wilk normality test

data:  smoker$weight
W = 0.89491, p-value = 0.0003276



	Shapiro-Wilk normality test

data:  nonsmoker$weight
W = 0.92374, p-value = 2.234e-05


Sin embargo, dado que el tamaño de cada grupo es mayor que 30 se puede considerar que el t-test sigue siendo suficientemente robusto, aunque es necesario mencionarlo en las conclusiones. Un test no paramétrico basado en la mediana (Mann-Withney-Wilcoxon test) o un test de Bootstraping serían más adecuados.


#### CONSIDERANDOQUELAS DOS POBLACIONES SON NORMALES (VERIFICAR CON PRUEBAS NO PARAMÉTRICAS)

#### 1. Parámetro Poblacional

El parámetro poblacional a contrastar es $\mu_1 < \mu_2$ 


#### 2. Hipótesis

Considérese las medias poblacionales $\mu_1$  $\mu_2$, con $\Delta_o =0$ 

\begin{equation}
H_{0}: \mu_1 - \mu_2 = \Delta_o \hspace{0.9cm} vs \hspace{0.9cm} H_{1}: \mu_1 - \mu_2 < \Delta_o
\end{equation}


#### 3. Estadística de prueba


Cuando la distribuciones poblacional es Normal, la variable estandarizada

\begin{equation}
Z=\frac{(\overline{X}_1-\overline{X}_2)-\Delta_o }{\sqrt{\frac{S^2_1}{ n_1}  +  \frac{S^2_2}{ n_2}}}\hspace{0.4cm} \hspace{0.4cm}  \hspace{1.2cm} ;  \hspace{1.2cm} Z\mid _{H_0} \sim Z(0,1)
\end{equation}

Verificando el resultado:

In [13]:
n1 <-length(smoker$weight)
n2 <-length(nonsmoker$weight)

In [14]:
Z<-(mean(smoker$weight)-mean(nonsmoker$weight) )  /sqrt(var(smoker$weight)/n1 + var(smoker$weight)/n2 )  ; Z

[1] -1.447518

Buscando el cuantil teórico:

In [15]:
########################################################################################################################
###########       NORMAL ESTANDAR       qnorm(p, mean = 0, sd = 1, lower.tail = TRUE, log.p = FALSE)   #################
########################################################################################################################

#conociendo z necesario

Z_alpha <- qnorm(0.1, 0 , 1, lower.tail = TRUE) # lower.tail=TRUE ent P[X <= x]  #alpha/2 para prueba dos colas,
Z_alpha                        


[1] -1.281552

$$Z=-1.44$$
$$Z_{0.1}=-1.28$$

#### c) Desición

 $ \vartheta :$ Luego como $Z= -1.44$, se tiene que a un nivel de significancia de $ \alpha= 0.05 $, $\;\;$ $Z= -1.44 < Z_{0.1}=-1.28 $ con lo cual se Rechaza $H_o$ concluyendose que SI existe evidencia estadística para poder afirmar que  el peso promedio de niños nacidos de madres fumadores sea menor que el peso de los niños de madres no fumadoras.

---
#### d) Prueba en R junto con intervalos de confianza

In [17]:
library(BSDA)

z.test(x=smoker$weight,
       y=nonsmoker$weight,
       alternative="less",
       mu=0,
       sigma.x=sd(smoker$weight),
       sigma.y=sd(nonsmoker$weight),
       conf.level=0.95)
#alternative="greater"   or   "less"     or   "two.sided"  


	Two-sample z-Test

data:  smoker$weight and nonsmoker$weight
z = -1.4967, p-value = 0.06723
alternative hypothesis: true difference in means is less than 0
95 percent confidence interval:
         NA 0.03964095
sample estimates:
mean of x mean of y 
   6.7790    7.1795 


In [ ]:
t.test(
  x           = smoker$weight,
  y           = nonsmoker$weight,
  alternative = "less",
  mu          = 0,
  var.equal   = TRUE,
  conf.level  = 0.95
        )#alternative="greater"   or   "less"     or   "two.sided"  